Hyper/Hyper.csproj

```diff
@@ -5,6 +5,11 @@
     <TargetFramework>net7.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
is this used? When i run the app (not debug) i still see bounding boxes

Reply:
no bounding boxes on release for me
![image](https://github.com/Non-Euclidean-World/Hyper/assets/54417806/4875cb8b-e8d9-4dcb-a4f4-7cfbda48f935)

Reply:
After i changed Configuration to release it worked. I thought that choosing run not debug was enought to have it run on release